### ISSUE TITLE
No redirection to project space from notification

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/web#10397]
ocis 5.1.0-prealpha+17b14ee78
web 8.0.0-rc.1

Steps:

- admin creates space
- admin adds member eintein 
- einstein open notification and click by space name

Actual: no redirect to Project space page. only after refreshing page

https://github.com/owncloud/web/assets/84779829/82dcec74-7d8f-4863-be20-f7f1a95abcb5

**Comment by JammingBen** [comment]
Good catch, a redirect to the space will only work after a page reload because then the space is being loaded by the runtime. In order to achieve that without a page reload we need to:

* Check if the notification subject is of type "space shared" (or something like that)
* If so, load the space by its id and add it to the list of loaded spaces (spaces store)

**Comment by LukasHirt** [comment]
Could not reproduce with `ownCloud Web UI 11.3.1`. Must have been solved in the meantime.
